app-server/app/src/main/scala/RfJsonProtocols.scala

```diff
@@ -0,0 +1,34 @@
+package com.azavea.rf
+
+import java.util.UUID
+import java.sql.Timestamp
+import java.time.Instant
+
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
+import spray.json._
+
+import com.azavea.rf.datamodel.latest.schema.tables.UsersRow
+import com.azavea.rf.datamodel.latest.schema.tables.OrganizationsRow
+
+trait RfJsonProtocols extends SprayJsonSupport with DefaultJsonProtocol {
+
+  implicit object UuidJsonFormat extends RootJsonFormat[UUID] {
+    def write(obj: UUID) = JsString(obj.toString)
+    def read(value: JsValue) = value match {
+      case JsString(obj) => UUID.fromString(obj)
+      case obj           => deserializationError("Expected UUID but got " + obj)
```

Comment:
Should also use string interpolation instead of concatenation, here and elsewhere.